CHANGELOG.md

```diff
@@ -16,6 +16,8 @@ how a consumer would use the library (e.g. adding unit tests, updating documenta
 
 - New command `code42 users remove-role` to remove a user role from a single user.
 
+- New command 'code42 users list --include-legal-hold-membership' to print the legal hold matter name and ID for any user on legal hold
```

Comment:
I know this has probably already been said, but what is the use-case for this functionality? I had heard most customers have a single matter for legal hold.

Is this flag general enough or this just for a specific use-case? I am also wondering if this is the right place to include this flag... What if I wanted to filter users by matter ID instead? Or what if I want to know which detection list or alert rules each user belong to as well?

Reply:
The primary use case that this addresses is "show me all the users on legal hold." 

There are customers that have a lot of legal hold matters. Therefore, the `legal-hold show <matter ID>` command would not meet their needs if they need to see users on all legal holds.

Also, `legal-hold show <matter ID>` produces the users in column format. This makes sense for a `show` command, which is meant to be a high-level overview (This was discussed in the [PR](https://github.com/code42/code42cli/pull/192#discussion_r562810149) that added the `devices list --include-legal-hold-membership`).  However, for an admin to work with the data and put it in different formats, the `list` command provided some more flexibility (at least for device info)

Reply:
I'm good with this being in the CLI proper. As it could be useful to any customer using legal hold. 

I think I'd also be ok with future flags for including if a user is on a detection list as well. Having the CLI `users` command be a kind of one-stop reporting tool to get all users info makes sense. Especially for larger customers who might have multiple teams (or automations) managing who gets put on what list/hold.